src/Talis/Persona/Client/Base.php

```diff
@@ -144,6 +152,7 @@ protected function checkConfig($config){
         }
 
         $requiredProperties = array(
+            'userAgent',
```

Comment:
`userAgent` is camelCase, whereas the other required properties are `snake_case`.  This seems confusing.